chaostoolkit/check.py

```diff
@@ -26,14 +26,16 @@ def check_newer_version(command: str):
             payload = r.json()
             latest_version = payload["version"]
             if payload.get("up_to_date") is False:
-                options = '--pre' if 'rc' in latest_version else ''
+                options = filter(None, [
```

Comment:
While I appreciate one liners, I don't think that reads as simply as: 

```
options = '--pre -U' if 'rc' in latest_version else '-U'
```

or I may be missing something?

Reply:
Nope you're right, code was intended to support multiple options (that can be added later on) but that made the code less clear/concise. 
i'll change it.

Reply:
Thanks!